ETC.tsv

```diff
@@ -9802,7 +9802,7 @@ ETC_20150323_009803	{memo X}Use multiple skills to suppress overkill.
 ETC_20150323_009804	{memo X}Use Status Points obtained through leveling up {nl}    and upgrade your character's abilities.
 ETC_20150323_009805	Elementalist
 ETC_20150323_009806	Vinas Camp
-ETC_20150323_009807	Rishkio City
+ETC_20150323_009807	Rieszhkio City
```

Comment:
I asked Xenobius about it. He said:

"Ryškus City, then if you want it to be a "Bright City"
And "sz" is a double consonant used in Polish/Czech languages"